components/Utils/Footer/SecurityModal/index.tsx

```diff
@@ -1,15 +1,20 @@
-import React from 'react'
+/** @jsx jsx */
+import { jsx } from '@emotion/core'
```

Comment:
y u importing the jsx if u not using emotion for this modal

Reply:
as with all the footer modals

Reply:
Bc it fixed the react thing that was complaining and stopping my commits from going through